sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy

```diff
@@ -1774,4 +1778,135 @@ class BlockBlobAPITest extends APISpec {
         then:
         thrown(IllegalArgumentException)
     }
+
+    @Unroll
+    def "Schedule Deletion min"() {
+        given:
+        def containerClient = dataLakeBlobServiceClient.createBlobContainer(getContainerName())
+        def blobClient = containerClient.getBlobClient(getBlobName())
+        blobClient.upload(defaultInputStream.get(), defaultDataSize)
+
+        when:
+        blobClient.getBlockBlobClient().scheduleDeletion(options)
+        def expiryTimeProperty = blobClient.getProperties().getExpiresOn()
+        def expiryTimeListProperty = containerClient
+            .listBlobs(new ListBlobsOptions().setPrefix(blobClient.getBlobName()), null)
+            .first()
+            .getProperties()
+            .getExpiresOn()
+
+        then:
+        (expiryTimeProperty != null) == hasExpiry
```

Comment:
I assume this is groovy syntax and expands to something like `expiryTimeProperty == null || expiryTimeProperty == hasExpiry`?